packages/react-inbox/src/components/Message/index.tsx

```diff
@@ -64,19 +64,20 @@ const Message: React.FunctionComponent<MessageProps> = ({
   const showMarkAsRead = !read && readTrackingId;
   const showMarkAsUnread = read && unreadTrackingId;
   const actions = useMemo(
-    () => [
-      {
-        href: data?.clickAction,
-        label: "View Details",
-        onClick: () => {
-          if (trackingIds?.clickTrackingId) {
-            trackEvent({
-              trackingId: trackingIds?.clickTrackingId,
-            });
-          }
+    () =>
+      [
+        data?.clickAction && {
```

Comment:
Conditionally rendering this button. If there is no `clickAction` the button does nothing